Slim/Plugin/Podcast/Plugin.pm

```diff
@@ -119,43 +163,39 @@ sub recentHandler {
 	my @menu;
 
 	foreach my $item(reverse values %recentlyPlayed) {
-		my $entry;
-		my $position = $cache->get("podcast-$item->{url}");
-
-		if ( $position && $position < $item->{duration} - 15 ) {
-
-			$position = Slim::Utils::DateTime::timeFormat($position);
+		my $from = $cache->get("podcast-$item->{url}");
```

Comment:
I'd prefer to have this `'podcast-' . $item->{url}` - it's less likely to cause a failure due to a typo (missing `>`), and easier to read. A detail.

Reply:
Didn't you use `podcast-$url` for the artwork before? But here it's expected to be the position within the entry?

Reply:
> Didn't you use `podcast-$url` for the artwork before? But here it's expected to be the position within the entry?
>
Yes but that's the RSS artwork, so there should be no conflict. I can add a namespace, but is this really necessary?

Reply:
There _should_ be no conflict. But I don't like using the same kind of key for different things, depending on the URL.